schema/settings.json

```diff
@@ -59,6 +59,12 @@
       "description": "Show the formatted date string in the given format. For example \"yyy-MM-dd HH:mm:ss\" results in \"2023-02-28 14:25:57 \", \"dd/MM/YYY HH:mm:ss (O)\" results in \"28/02/2023 14:25:57 (GMT +3)\". Find out more on unicode date field symbols on this link [https://www.unicode.org/reports/tr35/tr35-dates.html#Date_Field_Symbol_Table]",
       "default": "yyy-MM-dd HH:mm:ss",
       "pattern": "[yGuqQMLwdeEcabBhHkKmsSzOxX\\W].*"
+    },
+    "showExecutionsPerSecond": {
+      "type": "boolean",
+      "title": "Show Executions Per Second",
+      "description": "Show the executions per second calculated from the last execution time.",
```

Comment:
I think this should be explicit that number of executions is always one.

Reply:
I have changed this to explicitly say Cell Executions Per Second, but I understand it might not resolve the issue completely.

I'm in favour of outputs/time or updates/time (keeping it language-agnostic). 

@krassowski can it be done?

Reply:
@krassowski is there any exposed listener/hook I can use? (for writes to output?)

Reply:
There is a [`outputLengthChanged`](https://jupyterlab.readthedocs.io/en/latest/api/classes/outputarea.OutputArea-1.html#outputLengthChanged) signal in output area which you should be able to access via `cell.outputArea.outputLengthChanged`.